### PR TITLE
Add Status card

### PR DESCRIPTION
**Pull request by rawagner**
status

### CODE REVIEW

File: frontend/public/components/dashboards-page/overview-dashboard/status-card.tsx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+  const alertsResponse = alertsResults.getIn([ALERTS_KEY, 'data']) as PrometheusRulesResponse;
+  const alertsResponseError = alertsResults.getIn([ALERTS_KEY, 'loadError']);
+  const alerts = getAlerts(alertsResponse);
+
+  const cv = _.get(resources.cv, 'data') as ClusterVersionKind;
+  const LinkComponent = React.useCallback(() => {
+    console.log('abc');
+    return (
+      <Button bsStyle="link" onClick={() => clusterUpdateModal({cv})}>
+        Update
```
Comment (andybraren) [inline frontend/public/components/dashboards-page/overview-dashboard/status-card.tsx]:
```suggestion
        View details
```

Comment (rawagner) [reply frontend/public/components/dashboards-page/overview-dashboard/status-card.tsx]: updated

**Comment by rawagner** [comment]
Added Control Plane health

status_control

**Comment by andybraren** [comment]
(Still) looking great! Good call on "API Request Success Rate" - I see that's the label that the old Cluster Status page used too. Does etcd have a health endpoint that we'd be able to expose or calculate (assuming it's important)?

**Comment by rawagner** [comment]
> (Still) looking great! Good call on "API Request Success Rate" - I see that's the label that the old Cluster Status page used too. Does etcd have a health endpoint that we'd be able to expose or calculate (assuming it's important)?

not sure yet, I will be exploring that next week

**Comment by rawagner** [comment]
@kyoto any idea on a metric which could be used to get health of etcd ? https://github.com/openshift/console/pull/2806#issuecomment-535956405

File: frontend/public/components/dashboard/status-card/status-body.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { EmptyState, EmptyStateVariant, EmptyStateIcon, EmptyStateBody } from '@patternfly/react-core';
+import { CheckCircleIcon, QuestionIcon } from '@patternfly/react-icons';
+
+export const StatusBody = ({ children }) => (
```
Comment (vojtechszocs) [inline frontend/public/components/dashboard/status-card/status-body.tsx]: Please inline.

File: frontend/packages/console-plugin-sdk/src/typings/dashboards.ts [hunk 3]
```diff
@@ -13,10 +13,10 @@ import { PrometheusResponse } from '@console/internal/components/graphs';
 import { Extension } from './extension';
 import { LazyLoader } from './types';
 
-namespace ExtensionProperties {
+export namespace ExtensionProperties {
```
Comment (vojtechszocs) [inline frontend/packages/console-plugin-sdk/src/typings/dashboards.ts]: Can't we use `DashboardsOverviewHealthSubsystem['properties']` ?

**Comment by kyoto** [comment]
> @kyoto any idea on a metric which could be used to get health of etcd ? [#2806 (comment)](https://github.com/openshift/console/pull/2806#issuecomment-535956405)

Sorry, not sure. Probably best to ask someone on the etcd team.

**Pull request merged**